cmake/windows/resources/installer-Windows.iss.in

```diff
@@ -78,19 +78,27 @@ end;
 // Remove old OBS-NDI conflicting plugin - by Trouffman for DistroA https://github.com/DistroAV/DistroAV/
 
 // CAREFUL : this target the old AppID.
-const UninstallRegisterPath = 'Software\Microsoft\Windows\CurrentVersion\Uninstall\{A9039243-4FE7-45E7-8B11-7DC1ACB67B9D}_is1';
+const UninstallRegisteryPath411up = 'Software\Microsoft\Windows\CurrentVersion\Uninstall\{A9039243-4FE7-45E7-8B11-7DC1ACB67B9D}_is1';
+const UninstallRegisteryPath410 = 'Software\Microsoft\Windows\CurrentVersion\Uninstall\{CD703FE5-1F2C-4837-BD3D-DD840D83C3E3}_is1';
 
 function GetOldAppUninstallerPath(Param: String): String;
 var
   UninstallerPathRegistry: String;
 // Param is required to be used in other sections.
 begin
   result := '';
-  // Check primary registry location
-  if ( RegQueryStringValue(HKLM, UninstallRegisterPath, 'UninstallString', UninstallerPathRegistry) ) then
+  // Check primary registry location version 4.11+
+  if ( RegQueryStringValue(HKLM, UninstallRegisteryPath411up, 'UninstallString', UninstallerPathRegistry) ) then
```

Comment:
Is it possible for there to be some install/update/uninstall bug and someone have both <= 4.10 and >= 4.11 installed?

Reply:
Not expected.
The way PASCAL script work is as follow:
* The first match of a If statement with "exit" that statement.

In our example:
- if someone has any post 4.11 installed -> this will be removed.
- if they do not have the 4.11 but >= 4.10 it will remove it.

Anyone before 4.10 is out of support for over a year an half, this is not supported and that user will have to manually deal with it. At this point there is jsu tso much that can be done :D 

If a user has both 4.10 and 4.11+  installed at a time (could be if installed in 2 different locations) The install will remove OBS NDI 4.11, skip any other. At launch the user will see the error that obs-ndi is installed : if it run the distroav installer again : this will remove 4.10.